index.bs

```diff
@@ -1302,10 +1311,14 @@ The Web IDL definition for the {{ReadableStreamBYOBReader}} class is given as fo
 interface ReadableStreamBYOBReader {
   constructor(ReadableStream stream);
 
-  Promise<ReadableStreamReadResult> read(ArrayBufferView view);
+  Promise<ReadableStreamReadResult> read(ArrayBufferView view, optional ReadableStreamBYOBReaderReadOptions options = {});
   undefined releaseLock();
 };
 ReadableStreamBYOBReader includes ReadableStreamGenericReader;
+
+dictionary ReadableStreamBYOBReaderReadOptions {
+  [EnforceRange] unsigned long long min;
```

Comment:
Hmm, good point. 🤔 

We currently have slightly different logic between the `min` and non-`min` cases. It would be much nicer to convert all non-`min` cases to have a default `min`, and then just have `min` everywhere.

I'll look into it.